src/fhirtypes/ContactDetail.ts

```diff
@@ -0,0 +1,24 @@
+import { Period } from './Identifier';
+
+/**
+ * Represents the FHIR R4 metadata type ContactDetail.
+ *
+ * @see {@link http://hl7.org/fhir/R4/metadatatypes.html#ContactDetail}
+ */
+export type ContactDetail = {
```

Comment:
When my PR #3 is merged, it will introduce a src/fhirtypes/index.ts file. This and the other new FHIR types introduced in this PR should be added to that file.